src/Reducers/snackbarReducer.ts

```diff
@@ -0,0 +1,23 @@
+import { SnackbarAction, SnackbarState } from "../types/snackbar";
+
+export const snackbarReducer = (
+  state: SnackbarState,
+  action: SnackbarAction
+): SnackbarState => {
+  switch (action.type) {
+    case "show-snackbar":
+      return {
+        ...state,
+        open: true,
+        message: action.payload.message,
+        severity: action.payload.severity || "info",
```

Comment:
The EsLint showed me a red error because i didnt provide default and null checks for this can we add them?
```suggestion
             message: action.payload?.message ?? "",
        severity: action.payload?.severity || SEVERITIES.INFO,
```